addons/mail/models/ir_attachment.py

```diff
@@ -57,5 +61,47 @@ def _attachment_format(self, commands=False):
                     'res_id': attachment.res_id,
                     'res_model': attachment.res_model,
                 })
-            res_list.append(res)
-        return res_list
+            attachments.append(res)
+        return attachments
+
+    def _get_attachment_data_from_url(self, url):
+        try:
+            data = {}
+            request_image = None
+            page = requests.get(url, timeout=10)
```

Comment:
From the documentation:
```
Optional. A number, or a tuple, indicating how many seconds to wait for the client to make a connection and/or send a response.
Default None which means the request will continue until the connection is closed
```
I set it to 10 sec, but I feel like 5 would do the job too. It just need to be set, otherwise it will run for too long.